week4/src/sign.js

```diff
@@ -0,0 +1,79 @@
+const signInput = document.querySelector(".sign-input");
+const passwordInput = document.querySelector(".sign-input[type='password']");
+const loginButton = document.querySelector(".cta");
+const errorMessage = document.createElement("p");
+
+//
+// 이메일 관리
+//
+
+errorMessage.classList.add("error-message");
+
+function isEmailValid(email) {
+  return email.includes("@") && email.includes(".");
+}
+
+function emailError() {
+  const emailValue = signInput.value.trim();
+
+  if (emailValue === "") {
+    errorMessage.textContent = "이메일을 입력해 주세요";
+    signInput.classList.add("input-border-red");
+    signInput.after(errorMessage);
+  } else if (!isEmailValid(emailValue)) {
+    errorMessage.textContent = "올바른 이메일 주소가 아닙니다";
+    signInput.classList.add("input-border-red");
+    signInput.after(errorMessage);
+  }
+}
```

Comment:
@JaeBeen95 

중복되는 코드를 리팩토링 해보시는건 어떨까요?

```suggestion
function emailError() {
  const emailValue = signInput.value.trim();

  if (emailValue === "") {
    errorMessage.textContent = "이메일을 입력해 주세요";
  } else if (!isEmailValid(emailValue)) {
    errorMessage.textContent = "올바른 이메일 주소가 아닙니다";
  }
  
      signInput.classList.add("input-border-red");
    signInput.after(errorMessage);
}
```


혹은 아래 코멘트를 참고해보시죠

https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/pull/204#discussion_r1463964651